features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -34,6 +34,7 @@ using namespace mbed_cellular_util;
 #include "CellularLog.h"
 
 // URCs should be handled fast, if you add debug traces within URC processing then you also need to increase this time
+#define PROCESS_URC_CALLBACK_TIME 1000
```

Comment:
This approach works - it's what `ATCmdHandler` used to do, but I think the approach `ATCmdHandler` now takes is better - no separate timeout, but rather a special rule: "if only checking for OOBs, check readability with zero timeout if we're not in the middle of a line".

Something to consider in future. Not sure how it fits with the current code - for `ATCmdHandler` `process_oob` is now just calling `recv(NULL)` to indicate "not waiting for anything in particular".

Reply:
I guess we could do that later to optimize (especially noisy LPUART) power consumption.